woocommerce/woocommerce-admin
# Make analytics tables use the site's date format setting

Fixes #3666

Note that the bug report only talks about the Orders page, however it is also present on the Downloads, Customers and Revenues pages.

I've pulled the site's date format setting in to `window.wcSettings` so that it is available to the analytics tables.

Eg. (using `F, j Y` date format):

![image](https://user-images.githubusercontent.com/224531/74395958-619f2980-4e5c-11ea-9498-c78652a3972d.png)

Note that this is localised:

![image](https://user-images.githubusercontent.com/224531/74396005-84314280-4e5c-11ea-8701-6e2f09369dcb.png)

### Detailed test instructions:

Base functionality:

- Go to `/wp-admin/options-general.php` and set the date format to something not `m/d/Y`
- Go to `/wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Forders` and see the date format in use

Localization:

- Go to `/wp-admin/options-general.php`, set the language to something different, and set the date format to something that includes the month name (eg. `j F Y`)
- Go to `/wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Forders` and see the localized month name

### Changelog Note:

Fix: Make analytics tables use the site's date format setting
